src/Sentry.Unity.Editor/ScriptableSentryUnityOptionsEditor.cs

```diff
@@ -38,6 +38,7 @@ public override void OnInspectorGUI()
             EditorGUILayout.TextField("Environment Override", options.EnvironmentOverride);
             EditorGUILayout.Toggle("Attach Stacktrace", options.AttachStacktrace);
             EditorGUILayout.Toggle("Attach Screenshot", options.AttachScreenshot);
+            EditorGUILayout.Toggle("Attach Hierarchy", options.AttachViewHierarchy);
```

Comment:
I'm trying to keep it as "Unity" as possible. Since "View Hierarchy" is something Sentry and we're using it to capture the Hierarchy in Unity I think it works?

Reply:
Makes sense, keep as is then.
https://docs.unity3d.com/Manual/Hierarchy.html
![image](https://user-images.githubusercontent.com/6349682/223091485-04b0ac18-8579-46c6-bcdc-ec594a2e54c4.png)